### ISSUE TITLE
Compile error: prte_version.h missing

### ISSUE THREAD

**Issue by tonycurtis** [openpmix/prrte#993]
Thank you for taking the time to submit an issue!

## Background information

`prte_version.h` can't be found in VPATH builds.  Compiles in-source-tree.

### What version of the PMIx Reference Server are you using? (e.g., v1.0, v2.1, git master @ hash, etc.)

github master @ 540f40d5328727ca89da09d88c8de59fa084c080

##### What version of PMIx are you using? (e.g., v1.2.5, v2.0.3, v2.1.0, git branch name and hash, etc.)

github master @ 10b03e44c443d5115af66dc0ce62f98bc37ac740

### Please describe the system on which you are running

* Operating system/version: CentOS 8
* Computer hardware: aarch64
* Network type: IB

-----------------------------

## Details of the problem

```
make[2]: Entering directory '/lustre/home/arcurtis/src/openpmix/prrte/build/src'
  CC       prted/libprrte_la-prte.lo
In file included from ../../prrte-git/src/prted/prte.c:102:
/lustre/home/arcurtis/src/openpmix/prrte/prrte-git/include/prte.h:19:10: fatal error: prte_version.h: No such file or directory
 #include "prte_version.h"
          ^~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by rhc54** [comment]
Well....don't do that! VPATH = EVIL. 😄 

I'll take a gander in a bit.

**Comment by rhc54** [comment]
Fixed by #995
